docs/user/monitor_experiments_with_tensorboard.md

```diff
@@ -0,0 +1,178 @@
+# Monitor Your Experiments with TensorBoard
+
+[TensorBoard](https://www.tensorflow.org/tensorboard) is a powerful tool that allows you to monitor and visualize an experiment. This section introduces how to set up TensorBoard logging in garage, and how to view/share your TensorBoard logs.
+
+## Prerequisite
+The TensorBoard package should be installed upon garage's installation. It can be also manually installed via:
+```bash
+$ pip install tensorboard
+```
+
+## Logger Introduction and Setup
+Logging in garage is done via the [dowel](https://github.com/rlworkgroup/dowel) module.
+
+To start using dowel, import `logger`, dowel's logging facility:
+
+`from dowel import logger`
+
+`logger` takes in many different types of input and directs them to the correct output.
+
+Let's start by adding some outputs to logger:
+
+```bash
```

Comment:
so this is done for you if you use `@wrap_experiment` -- no need to recapitulate the dowel documentation.

You might want to cover adding your own output inside an experiment function though (usage is the same).